genie-agent/src/main/java/com/netflix/genie/agent/execution/statemachine/actions/ResolveJobSpecificationAction.java

```diff
@@ -126,6 +126,15 @@ protected Events executeStateAction(final ExecutionContext executionContext) {
             executionContext.setCurrentJobStatus(JobStatus.RESOLVED);
         }
 
+        UserConsole.getLogger().info(
+            "Job request criteria resolved to command '{}' on cluster '{}'",
+            jobSpecification.getCommand().getId(),
```

Comment:
We may want to think of sending back the name with the job specification if we intend to do this. It's distinctly possible that these ID's are pretty non-sensical for end users and would be better to have `on command {name} (id: {id})` as the log message or something